environment/templates/explorer-deployment.yml

```diff
@@ -0,0 +1,23 @@
+objectmeta:
+  name: explorer-mock
+spec:
+  selector:
+    matchlabels:
+      app: explorer-mock
+  template:
+    objectmeta:
+      labels:
+        app: explorer-mock
+    spec:
+      containers:
```

Comment:
Do we want to add request/limit values into this spec for the cpu and memory so we avoid adding another unbound deployment?

Reply:
Nice catch, will figure out some limits for it.

Reply:
Currently, we have issues with proper YAML unmarshalling of manifests, think we need to settle them down first.

Reply:
Done